integration_tests/integration/recommendations/ppcs/indeterminateSentences/sentenceToCommitIndeterminate.spec.ts

```diff
@@ -0,0 +1,173 @@
+import { fakerEN_GB as faker } from '@faker-js/faker'
+import searchMappedUserResponse from '../../../../../api/responses/searchMappedUsers.json'
+import searchActiveUsersResponse from '../../../../../api/responses/ppudSearchActiveUsers.json'
+import { RecommendationResponseGenerator } from '../../../../../data/recommendations/recommendationGenerator'
+import { CUSTODY_GROUP } from '../../../../../server/@types/make-recall-decision-api/models/ppud/CustodyGroup'
+import { RECOMMENDATION_STATUS } from '../../../../../server/middleware/recommendationStatus'
+import { SummaryList, testSummaryList } from '../../../../componentTests/summaryList.tests'
+import { formatPpudSentenceLength } from '../../../../../server/utils/dates/ppudSentenceLength/formatting'
+import { ppcsPaths } from '../../../../../server/routes/paths/ppcs'
+import { formatDateTimeFromIsoString } from '../../../../../server/utils/dates/formatting'
+
+describe('Indeterminate sentence - Your recall booking page', () => {
+  context('When viewing the sentence data', () => {
+    // Log in as PPCS user
+    beforeEach(() => {
+      cy.task('searchMappedUsers', { statusCode: 200, response: searchMappedUserResponse })
+      cy.task('ppudSearchActiveUsers', { statusCode: 200, response: searchActiveUsersResponse })
+      cy.signIn({ roles: ['ROLE_MAKE_RECALL_DECISION_PPCS'] })
+    })
+
+    const recommendationId = '123'
+    const testPageUrl = `/recommendations/${recommendationId}/sentence-to-commit-indeterminate`
+
+    const sentenceId = faker.number.int().toString()
+    const releaseDate = faker.date.future()
+
+    const defaultPPCSStatusResponse = [{ name: RECOMMENDATION_STATUS.SENT_TO_PPCS, active: true }]
+
+    context("and the sentence data hasn't been edited", () => {
+      const defaultRecommendationResponse = RecommendationResponseGenerator.generate({
+        bookRecallToPpud: {
+          custodyGroup: CUSTODY_GROUP.INDETERMINATE,
+          ppudSentenceId: sentenceId,
+        },
+        ppudOffender: {
+          sentences: [{ id: sentenceId, custodyType: 'Indeterminate', releaseDate }],
+        },
+      })
+      const ppudSentence = defaultRecommendationResponse.ppudOffender.sentences.find(
+        s => s.id === defaultRecommendationResponse.bookRecallToPpud.ppudSentenceId
+      )
+      beforeEach(() => {
+        cy.task('getRecommendation', { statusCode: 200, response: defaultRecommendationResponse })
+        cy.task('getStatuses', { statusCode: 200, response: defaultPPCSStatusResponse })
+      })
+      it('displays the default recommendation data', () => {
+        const summaryListContent = {
+          rows: [
+            { key: 'Custody type', value: defaultRecommendationResponse.bookRecallToPpud.custodyGroup },
```

Comment:
This should be selecting the custodyType, not the custodyGroup.

Reply:
@ppmg if I change it to custodyType it returns `undefined` 

In ppudSentenceGenerator.ts there's a PpudSentenceOptions type that has custodyType on it, but that doesn't include Indeterminate which is why I added it initially.

There's this enum that suggests the group is determinate/indeterminate:

`export enum CUSTODY_GROUP {
  DETERMINATE = 'Determinate',
  INDETERMINATE = 'Indeterminate',
}
`

So I'm wondering if actually it _should_ be custodyGroup, but that `PpudSentenceOptions` should have the custodyGroup instead of type? Or have we just been inconsistent with our naming?

Because the determinateCustodyTypes are _determinate_, _eds_ and _eds (non-parole)_ but I wouldn't expect _determinate_ to appear in that list if it's the umbrella group?

Reply:
Ah, I think we're displaying the wrong information in the controller. We're using CustodyGroup but looking at the designs it should be CustodyType as the example is a type, not a group.